### ISSUE TITLE
[feature request] Using a manifest.json file found in curseforge modpacks.

### ISSUE THREAD

**Issue by Sshmoob** [Miniontoby/ModUpdater#1]
Using the Curseforge app you can export a profile that can be uploaded as modpacks onto curseforge. These exported profiles contain a manifest.json that contains the same data that is in a mods.json that your program uses, except it has a bit more information. It doesn't have the names of the mods though. I made a small manifest.json [here](https://pastebin.com/ixrK0GUk) if you want to see what they look like.
I think you should support these because it means that we don't need to make a mods.json ourselves and instead the curseforge app can do it for us. It would also make your program support curseforge modpacks that already exist.

**Comment by Miniontoby** [comment]
Sure, I will look at it. I did the names thing so you can easly know what
is what and tbh I think it doesn't really get used... but maybe only in the
shell version

Also the fileIDs will just make it to not be an modupdater... even though
this is not actually a updater, but just an installer with updater
feature...

But I will look at it and see what I can do.

Op di 28 jun. 2022 19:58 schreef StormDragon_64 ***@***.***>:

**Comment by Sshmoob** [comment]
Hey, I was just wondering, are you almost done with this?

**Comment by Miniontoby** [comment]
I have looked at it, but not yet really done anything, since I have the
prior to make the project working FIRST

Also I am going on a holiday for 10 day today, so I don't think I will do a
lot but maybe a little bit...

Have a nice day,
Miniontoby

Op vr 8 jul. 2022 01:12 schreef StormDragon_64 ***@***.***>:

**Comment by Miniontoby** [comment]
So I have been thinking and looking at it.

So the manifest thing just has a fileID field which would would be fine with a ModInstaller what this thing actually mostly is.

So I think it would fit, since yeah the name I chose is wrong, since I forgot to change it when I created the github repo, because it started as modupdater thing for my own server, since downloading all the mods by myself was a lot of work...

But yeah I think I might add it, since it doesn't require to get the newest fileID for the ProjectID.
